voter/management/commands/voter_process_snapshot.py

```diff
@@ -48,23 +45,38 @@ def find_md5(row_data, exclude=[]):
 
 
 def get_file_lines(filename):
-    # ftfy lets us iterate over lines while it corrects encoding problems
-    lines = ftfy.fix_file(filename)
-
+    # guess the number of lines
+    f = open(filename, 'rb')
+    chunk = f.read(1024 * 1024)
+    newlines_per_meg = chunk.count(b'\n')
+    file_megs = os.stat(filename).st_size / (1024 * 1024)
+    line_count = file_megs * newlines_per_meg
+
+    # UTF16 or (presumably) BOM-less UTF8
+    f.seek(0)
+    if f.read(2) == b'\xff\xfe':
+        encoding = 'utf16'
+    else:
+        encoding = 'latin1'
+
+    f = open(filename, encoding=encoding)
```

Comment:
Should we close the file before reopening? (or put the block above in a `with` context?)